### DISCUSSION TITLE
Use bootstrap class `.primary` on a row or a cell instead of whole table

### DISCUSSION THREAD

**Discussion by yoonghm** [quarto-dev/quarto-cli#7706] [Q&A] [answered]
### Description

We could use `.primary` class in table:

````
 Right | Left | Default | Center
:------|-----:|---------|:------:
   12  |  12  |    12   |    12
  123  |  123 |   123   |   123
    1  |    1 |     1   |     1
: Markdown table with `.primary` attribute {.primary}
````

![image](https://github.com/quarto-dev/quarto-cli/assets/4215991/0fb16ab9-dbef-4f38-b4e0-0d18861266e7)

Can `.primary` be used on a row or a on cell?

**Comment by mcanouil** [comment, marked as answer]
Quarto accepts HTML code for table using raw code blocks.
If you want advanced control, you have to leave markdown syntax and use the actual format syntax, such as HTML in your case.
